lib/test/t_SmoothedUniform_std.expout

```diff
@@ -0,0 +1,64 @@
+Testing class SmoothedUniform
+checkConstructorAndDestructor()
+checkCopyConstructor()
+streamObject(const T & anObject)
+class=SmoothedUniform name=SmoothedUniform dimension=1 a=-0.5 b=1.5 sigma=0.5
+streamObject(const T & anObject)
+class=SmoothedUniform name=SmoothedUniform dimension=1 a=-0.5 b=1.5 sigma=0.5
+areSameObjects(const T & firstObject, const T & secondObject)
+areDifferentObjects(const T & firstObject, const T & secondObject)
+Distribution class=SmoothedUniform name=SmoothedUniform dimension=1 a=-0.5 b=1.5 sigma=0.5
+Distribution SmoothedUniform(a = -0.5, b = 1.5, sigma = 0.5)
+Elliptical = true
+Continuous = true
+oneRealization=class=Point name=Unnamed dimension=1 values=[1.15307]
+oneSample first=class=Point name=Unnamed dimension=1 values=[1.3731] last=class=Point name=Unnamed dimension=1 values=[1.75997]
+mean=class=Point name=Unnamed dimension=1 values=[0.990093]
+covariance=class=CovarianceMatrix dimension=1 implementation=class=MatrixImplementation name=Unnamed rows=1 columns=1 values=[0.581443]
+Kolmogorov test for the generator, sample size=100 is rejected
+Kolmogorov test for the generator, sample size=1000 is rejected
```

Comment:
Maybe something wrong with getRealization ?

Reply:
@jschueller agreed. getRealization and getSample differ (missing a_ in getSample)